packages/discord.js/typings/index.d.ts

```diff
@@ -3231,44 +3231,21 @@ export class VoiceState extends Base {
   public edit(options: VoiceStateEditOptions): Promise<this>;
 }
 
-export class Webhook extends WebhookMixin() {
-  private constructor(client: Client<true>, data?: RawWebhookData);
+export class BaseWebhook extends WebhookMixin() {
```

Comment:
see here for previous comment. `export class BaseWebhook`
![image](https://user-images.githubusercontent.com/28943913/224866492-8dced268-8b24-4be8-b9c5-f5209198cba6.png)
but we have no export